### PR TITLE
[Tests] Reuse NetworkResources in the linker tests.

### PR DESCRIPTION
**Pull request by mandel-macaque**
The linking tests do use a number of endpoints.
As with PR https://github.com/xamarin/xamarin-macios/pull/7418 we want
to have all the endpoints in a single place to update them if the server
side does give us problems.

### CODE REVIEW

**Comment by mandel-macaque** [comment]
Interesting, I ran them locally and had not timeouts. I'll retry.

**Comment by mandel-macaque** [comment]
build

File: tests/linker/mac/LinkAnyTest.cs [hunk 1]
```diff
@@ -64,20 +65,18 @@ public void GetWebPageAsyncTest ()
 
 		[Test]
 		public void WebClientTest ()
-		{
-			var url = "http://www.microsoft.com";
+		{;
```
Comment (wachaudh) [inline tests/linker/mac/LinkAnyTest.cs]: syntax error?

File: tests/linker/ios/link sdk/PclTest.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 #endif
 using NUnit.Framework;
 
+
```
Comment (wachaudh) [inline tests/linker/ios/link sdk/PclTest.cs]: unnecessary new line

**Comment by mandel-macaque** [comment]
Known issue: https://github.com/xamarin/maccore/issues/581

**Pull request merged**